Lib/test/test_ntpath.py

```diff
@@ -270,6 +277,12 @@ def test_normpath(self):
         tester("ntpath.normpath('//server/share/../..')",  '\\\\server\\share\\')
         tester("ntpath.normpath('//server/share/../../')", '\\\\server\\share\\')
 
+        # gh-96290: don't normalize partial/invalid UNC drives
```

Comment:
An invalid UNC drive part is skipped instead of normalizing it as a rooted path, but it's still normalized to replace forward slashes with backslashes.

```suggestion
        # gh-96290: don't normalize partial/invalid UNC drives as rooted paths
```